### PR TITLE
Update topics documentation CIVIC-5577

### PR DESCRIPTION
**Pull request by janette**
Issue: CIVIC-5577

## Description

Updates for Topics documentation, converts file from .md to .rst

### CODE REVIEW

File: docs/components/topics.rst [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+3. Uncheck the box under "Enabled" for Topics
+4. Click "Save configuration"
+
+Adding new icons
+------------------------------------
+
+**3 Important notes:** 
+  * Only users with the Administrator role can add new icon fonts.
+  * The font can only be changed if there are **NO** default icon values in use. Only one icon font can be used at a time.
+  * The display of the icons in facets and the drop down menu will break.
```
Comment (dafeder) [inline docs/components/topics.rst]: This should really be in "known issues"

File: docs/components/topics.rst [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+5. Click "Save"
+6. Navigate to ``Structure > Taxonomy > Topics > Manage Fields > Icon``
+7. Select your font from the font dropdown in the Icon field settings section.
+8. Click "Save settings"
+
+
+Known Issues
+------------------------------------
+
+* When enabling this module on an existing DKAN site, the default topics may be added to the top-level navigation menu. If this happens, simply use the taxonomy UI to move the new terms under the "Topics" term. New topics you add should then be added under the Topics term in the main menu.
```
Comment (dafeder) [inline docs/components/topics.rst]: I haven't seen this happen in ages. Have you? If not let's remove

File: docs/components/topics.rst [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+* The module adds a main menu link for "Topics". If you want a different word in place of "Topics", you will need to modify the term from the taxonomy UI. This will cause the DKAN Topics feature to be overridden and make future upgrades more difficult.
```
Comment (dafeder) [inline docs/components/topics.rst]: Is this still true? I thought we got this out of features

File: docs/components/topics.rst [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+
+* To filter by topics on the Datasets/Search page, field_topics needs to be added to the dataset search index. This should happen automatically when you enable this module and revert all DKAN core `features <https://www.drupal.org/project/features>`_. However, in some cases this has been observed not to work. To fix edit the datasets index manually (`admin/config/search/search_api/index/datasets/fields`) and add field_topics.
+
+* Since this module adds a field to the Dataset content type, your search indexes will need to be rebuilt, whether or not the previous step (manually adding the field to the index) is necessary. After installing, clear your dataset index and re-index all items by doing one of the following:
```
Comment (dafeder) [inline docs/components/topics.rst]: This doesn't seem necessary anymore, Topics is no longer a new module

File: docs/components/topics.rst (hunk 3 above)
Comment (janette) [inline docs/components/topics.rst]: @dafeder if the menu title is changed, there are no overrides, but the icons will not display in the menu. If you use string overrides to change 'Topics' to something else, the menu works and the icons display. However, the facet title and form label remain 'Topics' so I will just make these clarifications under known issues and remove the old stuff.

Comment (dafeder) [inline docs/components/topics.rst]: Sounds good thanks

**Pull request merged**